nixos/modules/services/misc/snapper.nix

```diff
@@ -103,6 +103,18 @@ in
       '';
     };
 
+    persistentTimer = mkOption {
+      default = false;
+      type = types.bool;
+      example = true;
+      description = ''
+        Set the `persistentTimer` option for the
```

Comment:
nit: Should it be `Persistent`?  If so, this (and the one in the [borgbackup module](https://github.com/NixOS/nixpkgs/blob/00c95af548cbee2ec12a515ae409a65d651273f2/nixos/modules/services/backup/borgbackup.nix#L364)) can be fixed in a future PR.

Reply:
Addressed with https://github.com/NixOS/nixpkgs/pull/314099